DAOS-4233 debug: allow to limit log file footprint (#3405)
daos-stack/daos: doc/environ.md

default. This can be modified by setting this environment variable
("D_LOG_FILE=/tmp/daos_server").

### `D_LOG_SIZE`

DAOS debug logs (both server and client) have a 1GB file size limit by

> default. When this limit is reached, current log file is closed and

Reviewer: ```suggestion
default. When this limit is reached, the current log file is closed and
```

renamed with a .old suffix, and a new one is opened. This mechanism

> will repeat each time limit will be reached, meaning that available

Reviewer: ```suggestion
will repeat each time the limit is reached, meaning that available
```

saved log records could be found in both ${D_LOG_FILE}[.old] files.
This can be modified by setting this environment variable

> ("D_LOG_SIZE=536870912").

Reviewer: Is it worth mentioning that only the last generation of `.old` is kept such that anything older than `2*D_LOG_SIZE` is discarded and lost?  Just so people are not confused about the earliest log records being missing.

Author: Well, any details are welcome on the doc side, what is really limiting me in this domain is that I have to do it in English ;-)

Reviewer: Indeed.  I can most certainly appreciate that.  Your English is **far** better than my French.

default. This can be modified by setting this environment variable
("D_LOG_FILE=/tmp/daos_server").

### `D_LOG_SIZE`

DAOS debug logs (both server and client) have a 1GB file size limit by
default. When this limit is reached, the current log file is closed and
renamed with a .old suffix, and a new one is opened. This mechanism
will repeat each time the limit is reached, meaning that available
saved log records could be found in both ${D_LOG_FILE} and last
generation of ${D_LOG_FILE}.old files, to a maximum of the most recent
2*D_LOG_SIZE records.
This can be modified by setting this environment variable

> ("D_LOG_SIZE=536870912").

Reviewer: Should add a comment mentioning that human-readable sizes (i.e. 1G) can be used here?  Something like:

```suggestion
("D_LOG_SIZE=536870912").  Sizes can also be specified in human-readable form using `k`, `m`, `g`, `K`, `M`, and `G`.  The lower-case specifiers are base-10 multipliers and the upper case specifiers are base-2 multipliers.
```

Author: Ok, agree! BTW, I need to move this to doc/admin/env_variables.md as doc/environ.md has recently been removed
